project_description.md

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
move to `projects/javascript/<whatever you want to call it>.md`

Reply:
moved to appropriate location with name switch from "js-library" to "noobJS"